### DISCUSSION TITLE
Spatie Media Library Pro with Orchid?

### DISCUSSION THREAD

**Discussion by wsdnathan** [orchidsoftware/platform#1878] [Q&A]
Has anyone tried creating an Orchid Field and integrating with the Spatie Media Library Pro uploader Blade component?

If so wondering if anyone could share their experience with switching to this coming from Orchid Attachments, or even share their new Field, thank you.

**Comment by modmatrix** [comment]
Did you end up trying this? If so, how did it work out?

**Reply by wsdnathan** [reply]
No I did not because I looked further into how Orchid handles Attachments and other types of uploads.
